src/react-native/03-creating-a-new-app/creating-a-new-app.md

```diff
@@ -6,43 +6,161 @@
+
+### Setup 2
+
+Before we can create or execute any tests, run the following command to install the packages related to the React Native Testing Library:
+
+✏️ Run:
+
+```shell
+npm install --save-dev @testing-library/react-native@12 @types/jest@29
+```
+
+✏️ Update **jest.config.js** to be:
+
+@diff ../../../exercises/react-native/03-creating-a-new-app/01-solution/jest.config.js ../../../exercises/react-native/03-creating-a-new-app/02-solution/jest.config.js
+
+✏️ Create **jest-setup.ts** and update it to be:
+
+@sourceref ../../../exercises/react-native/03-creating-a-new-app/02-solution/jest-setup.ts
+
+✏️ Delete the unneeded generated files inside `__tests__`.
+
+✏️ Create **App.test.tsx** and update it to be:
+
+@sourceref ../../../exercises/react-native/03-creating-a-new-app/02-solution/App.test.tsx
+
+✏️ Replace the contents of **App.tsx** and update it to be:
+
+@sourceref ../../../exercises/react-native/03-creating-a-new-app/02-solution/App.tsx
+
+### Verify 2
+
+After completing the previous steps you will be able to run unit tests. Execute the command:
+
+```shell
+npm run test
+```
+
+and "Test Suites: 1 passed, 1 total" will be written to the console.
+
+## Objective 3: Cleanup the generated code and add helpers
+
+Before we begin adding any content, its a good idea to clean up generated files and add code quality tools.
+
+### Setup 3
+
+✏️ Run:
+
+```shell
+npm install --save-dev @bitovi/eslint-config depcheck
+```
+
+✏️ Update **package.json** to be:
+
+@sourceref ../../../exercises/react-native/03-creating-a-new-app/03-solution/package.json
+@highlight 6-10, only
+
+✏️ Create **.prettierrc.js** to be:
+
+@sourceref ../../../exercises/react-native/03-creating-a-new-app/03-solution/.prettierrc.js
+
+✏️ Create **.depcheckrc** to be:
+
+@sourceref ../../../exercises/react-native/03-creating-a-new-app/03-solution/.depcheckrc
+
+✏️ Update **.eslintrc.js** to be:
+
+@diff ../../../exercises/react-native/03-creating-a-new-app/02-solution/.eslintrc.js ../../../exercises/react-native/03-creating-a-new-app/03-solution/.eslintrc.js
+
+✏️ Update **metro.config.js** to be:
+
+@sourceref ../../../exercises/react-native/03-creating-a-new-app/03-solution/metro.config.js
+@highlight 1, only
 
-### Exercise 1
+✏️ Now that we have prettier and eslint set up, we need to make sure our code is following the new standards. If you installed the [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) and [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) VS Code plugins, all you have to do is open and save each file, and VS Code will make the necessary changes. Fix these files:
```

Comment:
This seems like a pain, can we tell them to run a command instead, especially because we can’t expect that everyone will have installed those plugins or will be using VS Code?